google-cloud-resourcemanager/src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java

```diff
@@ -27,7 +27,9 @@
  *
  * @see <a
  *     href="https://cloud.google.com/resource-manager/reference/rest/v1/ListAvailableOrgPolicyConstraintsResponse#Constraint">Constraint</a>
+ * @deprecated v3 GAPIC client of ResourceManager is now available
```

Comment:
For the best developer experience, let's try and find the associated model/rpc in the v3 client